schemas/2.0/language.xsd

```diff
@@ -57,6 +57,7 @@
 	  <xs:enumeration value="Turkish" />
 	  <xs:enumeration value="Vietnamese" />	  
 	  <xs:enumeration value="Georgian" />
+      <xs:enumeration value="Catalan" />
```

Comment:
Respect indentation

Reply:
I am aligning with the previous lines as the lines before Catalan is not.
![image](https://user-images.githubusercontent.com/49319959/91040262-3321b880-e60e-11ea-8b18-dce66c1f3ef3.png)

Reply:
my bad sorry. Can you align all of them? :)